frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -784,10 +790,21 @@ function VirtualKeyboard:init()
 end
 
 function VirtualKeyboard:getKeyboardLayout()
-    return G_reader_settings:readSetting("keyboard_layout") or G_reader_settings:readSetting("language")
+    local lang
+    if G_reader_settings:nilOrTrue("keyboard_remember_layout") or
+       G_reader_settings:isTrue("keyboard_force_current_layout") then
+        lang = G_reader_settings:readSetting("keyboard_layout")
```

Comment:
(A bit confusing when quick reading this: it looks like the 2nd line was after a then and misindented with the 3rd line.
Personnal preference I guess, but I wouldn't mind more indentation on the 2nd line, even if it makes the G_reader... no longer vertically aligned.)

Reply:
Fixed